common/share widget/reports/ally-1-fr.html

```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Assessment d'accessibilité #1 - Share widget",
+	"language": "fr",
+	"description": "Partielle. Évaluation des nouveaux critères de succès aux niveaux A et AA de WCAG 2.1 et WCAG 2.2",
+	"tag": "share",
+	"parentdir": "share",
+	"parentPage": "Share widget",
+	"parentPageURL": "share",
+	"altLangPage": "a11y-1-en.html",
+	"dateModified": "2023-07-06",
+	"layout": "assessment_wrote_en-fr",
+	"reportURL": "a11y-1.json"
```

Comment:
This report path is invalid
```suggestion
	"reportURL": "pre-ally-1.json"
```